lib/dl_formula/dl_formula/mutation/registry.py

```diff
@@ -0,0 +1,12 @@
+import typing as tp
+
+from dl_api_lib.query.registry import is_forkable_dialect
+from dl_formula.core.dialect import DialectCombo
+from dl_formula.mutation.lookup import LOOKUP_MUTATOR_REGISTRY
+
+
+def get_mutation_functions_names(dialect: DialectCombo) -> tp.List[str]:
```

Comment:
We tend to use just `list[str]` in type hints

Reply:
fixed